frontend/src/components/Room.tsx

```diff
@@ -252,6 +295,7 @@ class Room extends React.Component<Props, State> {
 
   render() {
     const { classes } = this.props;
+    const username = this.state.username && this.state.username;
```

Comment:
change this

Reply:
Removed the second this.state.username...